crt_portal/static/sass/custom/proform.scss

```diff
@@ -0,0 +1,7 @@
+.crt-portal_date_inputs_btn {
+    display: flex;
```

Comment:
You don't need `crt-portal_date_inputs_btn` since you can add `display-flex` as a class name. (`display-flex` is an USWDS class name that automatically applies the `display: flex` CSS property)

Reply:
Ahh I was looking for a utility class for flexbox in the project files. Thanks for linking the uswds docs I forgot to look through them as a reference well 👍